app/source/js/activities/licensesCertificationsForm.js

```diff
@@ -209,7 +209,7 @@ function ViewModel(app) {
             app.model.jobTitleLicenses.clearCache();
             app.model.userLicensesCertifications.clearCache();
             // Go out
-            app.successSave();
+            app.shell.goBack();
```

Comment:
Same problem as commented at *addressEditor.js*, and repeated on line 240 of this file.